student-library/src/main/java/tds/student/sql/data/ItemResponseUpdate.java

```diff
@@ -10,12 +10,18 @@
 
 import java.util.UUID;
 
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.ANY;
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.NONE;
+
 /**
  * @author temp_rreddy
  * 
  */
+@JsonAutoDetect(fieldVisibility = NONE)
```

Comment:
Jackson will, by default, serialize/deserialize FIELD values.  The \@JsonProperty annotations below force it to *also* serialize the annotated methods.  The end result is that if we don't turn off field serialization, a serialized object will have both "_page" and "Page" properties with duplicate values.
My gut tells me to fix all these models to reduce our network bandwidth... but in the name of deadlines, I'm only touching the three model objects I can personally validate atm.